app/assets/javascripts/i18n.js

```diff
@@ -497,6 +510,63 @@
     return message.replace(/_#\$#_/g, "$");
   };
 
+  // Not part of the public API
+  I18n.splitInterpolate = function(message, matches, options) {
+    var value
+    , placeholder
+    , name
+    , regex;
+
+    var buffer = [message];
```

Comment:
Is this double wrapping the split array or wrapping a string into an array with a single item. What array is the reduce below looping?